src/rfbzero/redox_flow_cell.py

```diff
@@ -153,17 +157,18 @@ def __init__(
         if not 0.0 < self.alpha_cls < 1.0 or not 0.0 < self.alpha_ncls < 1.0:
             raise ValueError("Alpha parameters must be between 0.0 and 1.0")
 
-        if not isinstance(self.n_cls, int) or not isinstance(self.n_ncls, int):
-            raise ValueError("'n_cls' and 'n_ncls' must be integers")
+        if not isinstance(self.num_electrons_cls, int) or not isinstance(self.num_electrons_ncls, int):
+            raise ValueError("'num_electrons_cls' and 'num_electrons_ncls' must be >= 1")
```

Comment:
The check and the error message don't match. Let's remove the isinstance check, just ensure >= 1, and split these into two checks:
```py
if self.num_electrons_cls < 1:
    raise ValueError("'num_electrons_cls' must be >= 1")

if self.num_electrons_ncls < 1:
    raise ValueError("'num_electrons_ncls' must be >= 1")
```

Reply:
forgot to do this, good catch.

Reply:
actually for this the value is only allowed to be an integer. User could input say 1.5 and all calculations would work, but it is physically impossible in real world.